.gitignore

```diff
@@ -0,0 +1,2 @@
+result*        # nix-build symlinks
```

Comment:
Just `result` is already ignored below and multiple (output) derivations are separated via dash, so there is no need to be redundant.

```suggestion
result-*        # nix-build symlinks
```

Reply:
Fixed!